psd-to-html-creative-agency/src/styles/_portfolio-section.scss

```diff
@@ -0,0 +1,138 @@
+.portfolio {
+  @include container(1240px, 20px);
+  margin-top: -60px;
+  padding-top: 90px;
+  padding-bottom: 20px;
+
+  @include tablet-up {
+    margin-top: 0;
+    padding-top: 60px;
+    padding-bottom: 100px;
+  }
+
+  @include laptop-up {
+    padding-top: 132px;
+    padding-bottom: 180px;
+  }
+}
+
+.portfolio__buttons {
+  display: none;
+
+  @include tablet-up {
+    @include flex(space-between);
+    margin-top: 78px;
+    width: 680px;
+  }
+
+  @include laptop-up {
+    width: 740px;
+  }
+}
+
+.projects-container {
+  @include tablet-up {
+    font-size: 1.24vw; // for sizing
```

Comment:
tokio nesu matęs :) Teksto dydis su `vw`?

Reply:
Sitoje situacijoje `font-size: ..vw` naudojamas proporcingam elemento` .projects-container` didinimui/mazinimui.
Elemento `height` ir `width` priklauso nuo `em` , taigi didejant ekrano plociui elemento aukstis ir plotis dideja proporcianaliai. Zinoma norint isvengti problemu kylanciu is "font -size inheritance" reikia nepamirsti elementu esanciu  `projects-container` ir naudojanciu  `em` dydzio vienetus.

Reply:
Čia dar trumpas straipsnis apie panaudojimą šrifto dydžio nustatymui [CSS Tricks](https://css-tricks.com/viewport-sized-typography/)

Reply:
Nice, tikrai įdomus reikalas. Naudoju `vw`ir `vh`dažnai, bet niekada teksto dydžiui nusakyti.